### PR TITLE
fix(docs): add guide for web component layouts

### PR DESCRIPTION
**Pull request by Kiarokh**
and explain how a component can span through more
columns or rows, within the slot.

### CODE REVIEW

**Comment by Kiarokh** [comment]
Not sure if the page has the best title, or whether it should be elsewhere

File: doc/concepts/lime-web-component-placement.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+name: Lime Web Component Placement & Layout
+route: /lime-web-component-placement-layout
+menu: Concepts
```
[the remark this answers is not in the document]
Comment (Kiarokh) [reply doc/concepts/lime-web-component-placement.mdx:4]: Me neither, I asked @BjornOstberg and @TommyLindh2 and they suggested this place. Is it easy for you to move it there @adrianschmidt ? I don't seem to be able to edit the docs there.

[the remark this answers is not in the document]
Comment (Kiarokh) [reply doc/concepts/lime-web-component-placement.mdx:4]: (or don't know how to edit)

**Comment by adrianschmidt** [comment]
Moved to Lundalogik/lime-crm#630

**Pull request closed without merging**